hack/vendor-console.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+STARTTIME=$(date +%s)
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+CONSOLE_REPO_PATH=${CONSOLE_REPO_PATH:-$OS_ROOT/../origin-web-console}
+if [ ! -d "$CONSOLE_REPO_PATH" ]; then
+  echo "The console repo at path $CONSOLE_REPO_PATH does not exist."
+  echo "Make sure you have cloned the web console repo locally:  git@github.com:openshift/origin-web-console.git"
+  echo "Or, you can specify a different path with the CONSOLE_REPO_PATH environment variable."
+  exit 1
+fi
+
+echo "Making sure go-bindata binary has been built..."
+pushd ${OS_ROOT}/Godeps/_workspace > /dev/null
+  godep_path=$(pwd)
+  pushd src/github.com/jteeuwen/go-bindata > /dev/null
+    GOPATH=$godep_path go install ./...
+  popd > /dev/null
+popd > /dev/null
+
+if [ -z ${GIT_REF+x} ]; then 
+  echo "No git ref specified, using current state of the repo including any unstaged changes...";
+else 
+  echo "Using git ref ${GIT_REF}, all untracked and uncommitted changes will be stashed..."
+  # TODO improve this to return the user back to the state they were in
+  pushd "${CONSOLE_REPO_PATH}" > /dev/null
+    git add .
+    git stash
+    git checkout $GIT_REF
+  popd > /dev/null
+fi
+
+echo "Building bindata.go files..."
+pushd "${OS_ROOT}" > /dev/null
+  # Put each component in its own go package for compilation performance
+  # Strip off the dist folder from each package to flatten the resulting directory structure
+  # Force timestamps to unify, and mode to 493 (0755)
+  Godeps/_workspace/bin/go-bindata -nocompress -nometadata -prefix "$CONSOLE_REPO_PATH/dist"      -pkg "assets" -o "pkg/assets/bindata.go"      -ignore "\\.gitignore" $CONSOLE_REPO_PATH/dist/...
+  Godeps/_workspace/bin/go-bindata -nocompress -nometadata -prefix "$CONSOLE_REPO_PATH/dist.java" -pkg "java"   -o "pkg/assets/java/bindata.go" -ignore "\\.gitignore" $CONSOLE_REPO_PATH/dist.java/...
```

Comment:
Is the ignore parameter right here? It looks like it's a regex of filenames. I don't think it will match anything under dist.

https://github.com/jteeuwen/go-bindata#usage